.github/workflows/build_and_test.yml

```diff
@@ -3,15 +3,7 @@ name: Build and test
 on:
   push:
     branches:
-    - branch-3.2
-  pull_request:
-    branches:
-    - branch-3.2
-  workflow_dispatch:
-    inputs:
-      target:
-        description: 'Target branch to run'
-        required: true
+      - '**'
```

Comment:
Well, is this correct?

The original one is the same with what we did in `branch-3.1`. Do you mean that `branch-3.1` is not working too?
- https://github.com/apache/spark/blob/branch-3.1/.github/workflows/build_and_test.yml

Reply:
I think `branch-3.1` is fine. We changed to distribute the tests to each forked repository after `3.1`, IIRC.